CHANGES.md

```diff
@@ -1,6 +1,6 @@
 # Changes
 
-## Unreleased
+## 0.27.1
 
 - Bump geo-types
```

Comment:
nit: maybe delete this line and update the pre-existing "update to geo-types" a few lines down. It's kind of confusing to have both entries in the change log.